policies/testing/README.md

```
@@ -0,0 +1,117 @@
+
+## Overview
+
+This policy enforces that tests are executed in CI and all tests pass. It is language-agnostic and works with any collector that writes to the normalized `.testing` paths (e.g., the `golang` collector for Go projects).
+
+## Policies
+
+This plugin provides the following policies (use `include` to select a subset):
+
+| Policy | Description | Failure Meaning |
+|--------|-------------|-----------------|
+| `executed` | Tests should be executed in CI | No test execution data found |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)